src/RecoveryChallengeController.js

```diff
@@ -15,7 +15,7 @@ import hbs from 'handlebars-inline-precompile';
 import Enums from 'util/Enums';
 import FormController from 'util/FormController';
 import FormType from 'util/FormType';
-import FooterSignout from 'views/shared/FooterSignout';
+import FooterMFA from 'views/shared/FooterMFA';
```

Comment:
Don't think we need to add MFA verify link for recovery flow (RecoveryChallenge, RecoveryQuestion).
Let's follow up with PM in Slack.

Reply:
Removed